commands/test/test_test.go

```diff
@@ -257,3 +257,34 @@ ENTRYPOINT ["java","-cp","app:app/lib/*","hello.Application"]`
 		t.Errorf("Dockerfile test failure. Got %v failures, expected %v", actual, expected)
 	}
 }
+
+func TestListFile(t *testing.T) {
+	testTable := []struct {
+		name       string
+		fileList   []string
+		shouldFail bool
+	}{
+		{
+			name:       "when flag exists it should use the flag value",
+			fileList:   []string{"testdata/deployment.yaml", ""},
+			shouldFail: false,
+		},
+	}
+
+	for _, testUnit := range testTable {
+		t.Run(testUnit.name, func(t *testing.T) {
+			viper.Set("policy", "testdata/policy/test_policy.rego")
+			exitCallCount := 0
+			cmd := NewTestCommand(func(int) {
+				exitCallCount += 1
+			}, func() OutputManager {
+				return new(FakeOutputManager)
+			})
+			cmd.Run(cmd, testUnit.fileList)
+
+			if testUnit.shouldFail == false && exitCallCount >= 1 {
+				t.Error("we did not expect to fail here, yet we did")
+			}
+		})
+	}
+}
```

Comment:
I'm personally not a fan of some of the tests we have that are needing to set viper configs (set applies globally, so the `policy` value changes for all tests). Additionally, validating whether or not the test passed based on the `OutputManager` and checking call counts.

Since this test really just calls `cmd.Run()` for the TestCommand, I'd argue this is the same test that you added in `acceptance.bats`, and we can just remove this one here.

Thoughts?

Reply:
That works for me. I'll drop.